*** pngset.c.orig	2007-07-17 15:16:44.000000000 -0800
--- pngset.c	2007-07-17 15:16:53.000000000 -0800
***************
*** 269,281 ****
     /* check for width and height valid values */
     if (width == 0 || height == 0)
        png_error(png_ptr, "Image width or height is zero in IHDR");
- #ifdef PNG_SET_USER_LIMITS_SUPPORTED
-    if (width > png_ptr->user_width_max || height > png_ptr->user_height_max)
-       png_error(png_ptr, "image size exceeds user limits in IHDR");
- #else
     if (width > PNG_USER_WIDTH_MAX || height > PNG_USER_HEIGHT_MAX)
        png_error(png_ptr, "image size exceeds user limits in IHDR");
- #endif
     if (width > PNG_UINT_31_MAX || height > PNG_UINT_31_MAX)
        png_error(png_ptr, "Invalid image size in IHDR");
     if ( width > (PNG_UINT_32_MAX
--- 269,276 ----
